group1/communication/node/main.js

```diff
@@ -251,13 +415,14 @@ var message_template = {
 var channels = {};
 var sockets = {};
 
-// WebSocket stuff
+// webrtc stuff begins
+
 io.on('connection', function(socket) {
-        socket.on('msg', function(msg) {
-                io.emit('msg', msg);
+        /*socket.on('msg', function(msg) {
```

Comment:
Lines 421-425 are all commented out.  Are these necessary anymore or can they be removed?